build-release.sh

```diff
@@ -172,6 +174,13 @@ if [ "${build_classical}" == "1" ]; then
 
   ## Javascript (Classical) ##
 
+  # Web (editor)
+  rm -rf ${tmpdir}/webtmp/
+  mkdir -p ${tmpdir}/webtmp/
```

Comment:
I guess you could unzip in `${webdir}/` directly and not need `${tmpdir}/webtmp/`?